tests/unittests/InterpreterTest.cpp

```diff
@@ -898,6 +898,89 @@ TEST(Interpreter, nonLinearClassifier) {
   }
 }
 
+/// Generate images in two classes. A "line" is labeled as 0
+/// and a "cross" is labeled as 1
+static void generateImageData(Tensor &images, Tensor &labels, unsigned numSamples, bool isTrain) {
```

Comment:
nit: numSamples can be infered from Tensors' shape: `numSamples = images.dims()[0]`